CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
+## v0.2.2 Singapore
+
+### Added
+- Solidity mappings can now have a named key and a named value types. [seanyoung](https://github.com/seanyoung)
+
+### Changed
+- Solang now uses LLVM 15. [LucasSte](https://github.com/LucasSte)
+- The Subtrate Scale encoding now happens during the compiler's code generation phase, instead of during the code 
+emission phase. [xermicus](https://github.com/xermicus)
+- Solidity on Solana now required the Anchor framework for the client code, and the `@solana/solidity.js`
+  Typescript library has been deprecated.
```

Comment:
It's more than deprecated: it no longer works because we don't generate the abi file.

Actually I was thinking of removing the npm library and marking the repo readonly.

Reply:
>  it no longer works.
This is the meaning of 'deprecate', according to the [dictionary](https://www.dictionary.com/browse/deprecate):

> Computers. to cease supporting or recommending the use of (older elements, features, or versions of software)

Reply:
None of the links you provided me explain the difference between deprecated and unavailable.

Reply:
I'll change to `phased out`.

Reply:
> Actually I was thinking of removing the npm library and marking the repo readonly.

That is a great idea. I was also thinking about that, but I would do it after we do a new release. Solang v0.2.1 still needs `@solana/solidity.js`.